source/php/Component/Icon/Icon.php

```diff
@@ -56,12 +56,13 @@ public function init()
 
         //Identify as an image
         $this->data['attributeList']['role'] = "img";
-        $this->data['attributeList']['aria-label'] = $this->getAltText($icon);
-        $this->data['attributeList']['alt'] = $this->getAltText($icon);
```

Comment:
Alt text is needed, due to the role "image".

Reply:
A "image" without an alt-text is not applicable.

Reply:
Im not sure about it. I used this as reference for the PR:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role